cpp/modmesh/view/RCameraController.cpp

```diff
@@ -295,9 +295,9 @@ void ROrbitCameraController::updateCameraPosition(const CameraInputState & input
     }
     else
     {
-        const float x = clamp(input.txAxisValue + (input.leftMouseButtonActive ? input.rxAxisValue : 0));
-        const float y = clamp(input.tyAxisValue + (input.leftMouseButtonActive ? input.ryAxisValue : 0));
-        const auto translation = QVector3D(x, y, input.tzAxisValue) * linearSpeed() * dt;
```

Comment:
removed check for `input.leftMouseButtonActive` because it's always false here. Because in the [if above](https://github.com/solvcon/modmesh/pull/408/commits/ee98cd1c5d4680f9c76ec5c98a45f71a989e7a88#diff-be9511fa234431d29c196ed57eaf68dbf67a702bc5400c605cdb9d5450a48d33L263), there's already a check for `input.leftMouseButtonActive`

Reply:
Code like this needs a lot of fine-tuning.  Thanks for doing this.  It's great.